AppController/lib/datastore_server.rb

```diff
@@ -69,6 +69,15 @@ def self.start(master_ip, db_local_ip, my_ip, table)
   def self.stop(table)
      MonitInterface.stop(:datastore_server)
   end
+  # The following are needed to comply to the djinn calling in
```

Comment:
That's correct at this time. If we do that, we may not be able to accept patches for different database though. I would argue that if we want to remove this ability (ie to use different DB) we should do so in a different pull request. This is just to fix an error during shutdown. What do you think?